.template.config/template.json

```diff
@@ -0,0 +1,15 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Jordi Sanchez",
```

Comment:
```suggestion
    "author": "Jordi Sánchez",
```
? :D Or maybe even:
```suggestion
    "author": "Jordi Sánchez García",
```

Reply:
Don't take it too seriously, I just like the special characters in your name ;)

Reply:
ohh man! What a reviewer are you!!
:-D